src/datasets/other/attitudes_daily/reference.md

```diff
@@ -16,7 +16,7 @@ Each question has an identifier in the `attitudes_daily` table as the `question_
 * All the sites I’ve visited recently have worked; none of them seem broken (`sites_work`)
 * Using the internet helped me meet my goals today (`met_goals`)
 
-User responses are processed and mapped to their `question_key`'s in [this script](https://github.com/mozilla/bigquery-etl/blob/master/templates/telemetry_derived/surveygizmo_daily_attitudes/import_responses.py), creating a table `moz-fx-data-shared-prod.external.survey_gizmo_daily_attitudes` which is subsequently joined to `clients_daily`.
+User responses are processed and mapped to their `question_key`'s in [this script](https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/surveygizmo_daily_attitudes/import_responses.py), creating a table `moz-fx-data-shared-prod.external.survey_gizmo_daily_attitudes` which is subsequently joined to `clients_daily`.
```

Comment:
This script doesn't seem to exist anymore

Reply:
Your new link works. I see the script there.

Reply:
Ah, I looked in the wrong directory